### ISSUE TITLE
Update to mdi/svg 4.3.95

### ISSUE THREAD

**Issue by smeijer** [levrik/mdi-react#65]
@mdi is currently at `4.3.95`, while the used version is `3.7.95`. Can you bump it? I'm missing some icons in this package.

**Comment by levrik** [comment]
Hey @smeijer. I know. The process of updating is quite annoying at the moment. Since I got an invitation to the GitHub Actions beta I'm experimenting with them to automate this process just right now.
See https://github.com/levrik/mdi-react/commit/15bda53be25538daad054979feb0587c4d67dd21/checks for a first run of the simple build process.
I hope that this is finished by the end of this weekend so it can be used to keep the icons up to date from there on.

**Comment by levrik** [comment]
`4.3.95` is included with `6.0.0`. The process is automated via GitHub Actions now so mdi updates should be on time in future.

**Comment by smeijer** [comment]
A little off-topic, but where you able to automate it based on activity in the `mdi/svg` repo? Or does it still require a manual action in this repo?

Thanks for the update!

**Comment by levrik** [comment]
Some explanation of the previous process:

I set up the Renovate bot to issue PRs if `@mdi/svg` updates. That is still the same now. Before I had to manually merge, update the changelog and readme and publish (building takes about 4 minutes). Also I had to publish two packages since this repo also publishes a version for Preact. It's not a that crazy process but it took time which often was missing.
Now I automated the process after manually merging the PR by Renovate.
It updates the changelog and readme and builds and publishes both packages.
The coming weekend I'll go even further and will let the GitHub Action run if Renovate creates a PR. This will then also include the merging in the automated process.
